tools/pagerduty-migrator/scripts/add_users_pagerduty_to_grafana.py

```diff
@@ -0,0 +1,44 @@
+import os
+import secrets
+import sys
+import requests
+
+from urllib.parse import urljoin
+from pdpyras import APISession
+
+PAGERDUTY_API_TOKEN = os.environ["PAGERDUTY_API_TOKEN"]
+PATH_USERS_GRAFANA = "/api/admin/users"
+URL_GRAFANA = os.environ["URL_GRAFANA"] # Example: http://localhost:3000
+USERNAME_GRAFANA = os.environ["USERNAME_GRAFANA"]
+PASSWORD_GRAFANA = os.environ["PASSWORD_GRAFANA"]
+SUCCESS_SIGN = "✅"
+ERROR_SIGN = "❌"
+
+def pd_list_users():
+
+    session = APISession(PAGERDUTY_API_TOKEN)
+
+    users = session.list_all("users", params={"include[]": "notification_rules"})
+
+    for user in users:
+        password = secrets.token_urlsafe(15)
+        username = user["email"].split("@")
+        json = {"name": user["name"], "email": user["email"], "login": username[0], "password": password}
+        post_grafana(json)
+
+def post_grafana(data):
```

Comment:
Probably change the name of the function to something like `create_grafana_user` to indicate it creates a Grafana user via a POST request?

Reply:
> Consider changing the name of the function to something like `create_grafana_user` to indicate it creates a Grafana user via a POST request?

In this commit, I changed the function name to follow the default.
https://github.com/grafana/oncall/pull/403/commits/a920062c8faea39f95348d131b5b016a8dff62ab